i18n/en-US.properties

```diff
@@ -16,6 +16,8 @@ be.activitySidebar.activityFeed.taskMissingError = This task no longer exists
 be.add = Add
 # Text to display when app is disabled by applied access policy
 be.additionalTab.blockedByShieldAccessPolicy = Use of this app has been disabled by the applied access policy
+# Error message when an annotation comment update fails
+be.api.annotationUpdateErrorMessage = This comment could not be modified.
```

Comment:
I may be confused, but do we update `messages.js` and let the `build:i18n` update the en-US.properties file?